src/vm/arm64/asmhelpers.asm

```diff
@@ -1106,9 +1106,9 @@ MainLoop
         
 Success         
         ldr     x13, =g_dispatch_cache_chain_success_counter
-        ldr     x9, [x13]
-        subs    x9, x9, #1
-        str     x9, [x13]
+        ldr     x16, [x13]
```

Comment:
Actually x9 needs to point to ResolveCacheElem  as long as its contents are required.
x9 is modified in this routine